Shared/Samples/Browse WFS layers/BrowseWFSLayersView.swift

```diff
@@ -0,0 +1,195 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+/// The initial view of the Browse WFS Layers sample.
+struct BrowseWFSLayersView: View {
+    /// The URL of the service to load.
+    @State private var serviceURL = URL(string: "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities")!
+    /// A Boolean value indicating whether the service view is pushed on the
+    /// navigation stack.
+    @State private var isServiceViewPresented = false
+    
+    var body: some View {
+        NavigationStack {
+            Form {
+                Section {
+                    TextField(
+                        "WFS Service",
+                        value: $serviceURL,
+                        format: .url,
+                        prompt: Text("WFS Service URL")
+                    )
+                    Button("Load") {
+                        isServiceViewPresented = true
+                    }
+                    .foregroundStyle(.accent)
+                    .containerRelativeFrame(.horizontal)
+                    .multilineTextAlignment(.center)
+                }
+            }
+            .navigationDestination(isPresented: $isServiceViewPresented) {
+                WFSServiceView(serviceURL: serviceURL)
+            }
+        }
+    }
+}
+
+/// A view that loads a WFS service and lists its layers.
+private struct WFSServiceView: View {
+    /// The URL of the WFS service to load.
+    let serviceURL: URL
+    
+    @Environment(\.dismiss) private var dismiss
+    
+    /// The loaded WFS service.
+    @State private var service: WFSService?
+    /// The error if the service failed to load, otherwise `nil`.
+    @State private var serviceLoadError: Error?
+    /// A Boolean value indicating whether the service failed to load.
+    @State private var serviceLoadDidFail = false
+    
+    var body: some View {
+        if let serviceInfo = service?.serviceInfo {
+            Form {
+                Section("Layers") {
+                    ForEach(serviceInfo.layerInfos, id: \.name) { layerInfo in
+                        NavigationLink(layerInfo.title) {
+                            WFSServiceLayerView(layerInfo: layerInfo)
+                        }
+                    }
+                }
+            }
+            .navigationTitle(serviceInfo.title)
+        } else {
+            ProgressView()
+                .navigationTitle("Loading Service")
```

Comment:
```suggestion
            ProgressView("Loading Service…")
```

When first using the sample, I was a little confused on what exactly was loading here since I did not think to look at the navigation title, so this seems a little clearer. That is just my opinion, though, so feel free to veto.

Reply:
The HIG [recommends](https://developer.apple.com/design/human-interface-guidelines/progress-indicators#macOS) (at least for macOS):

> **Avoid labeling a spinning progress indicator.** Because a spinner typically appears when people initiate a process, a label is usually unnecessary.

That's why I used a navigation title instead of a label.